### PR TITLE
Update blocked Farpoint Defense mission

### PR DESCRIPTION
**Pull request by tehhowch**
Refs #3424 and #3467

 - Adds a blocked check to the Quarg recruitment, on the premise that the empathetic Elias would offer medical supplies to help Farpoint's injured. This blocked check then requires the player to arrive at Farpoint with the needed 2 space free for the bomb.
 - Splits the log entry for the Quarg escorts (as some players choose instead to wander with them, and may forget exactly when they picked up these escorts).
 - Adds an `on enter` to Bomb Zenith 1, which hints at the alternate solution. The text is per @Amazinite's https://github.com/endless-sky/endless-sky/issues/3467#issuecomment-354374324, with a paragraph break added.

New dialog & conversation appearance:
new quarg 2
new bomb 1 offer
alternate on enter

### CODE REVIEW

File: data/wanderers middle.txt [hunk 1]
```diff
@@ -2083,11 +2087,10 @@ mission "Wanderers: Sestor: Bomb Zenith 1"
 	to offer
 		has "Wanderers: Sestor: Quarg Help 2: done"
 	cargo "unknown device" 2
-	blocked "You will need two tons of cargo space to take on the next mission for Danforth."
 	on offer
-		log "Recruited some Quarg warships to help defend human space from the Kor Sestor drones, which are now presumably under Alpha control. Danforth believes that he has located the Alpha base from which the drones are being controlled; if the base is destroyed the drones ought to become aimless and be easily defeated."
```
Comment (jafdy) [inline data/wanderers middle.txt]: Why this change?

File: data/wanderers middle.txt [hunk 2]
```diff
@@ -2027,14 +2027,18 @@ mission "Wanderers: Sestor: Quarg Help 2"
...
-			`	He leaves to try to find help, and returns about fifteen minutes later. "I've got four Quarg ships who volunteered to help," he says. "That may not seem like a lot, but their ships pack quite a punch. Good luck, captain."`
-			`	You thank him for his assistance, and hurry to fire up your ship and return to human space. Hopefully Danforth's base has not already been overrun in your absence.`
+			`	He leaves to try to find help, and returns about fifteen minutes later. "I've got four Quarg ships who volunteered to help," he says. "That may not seem like a lot, but their ships pack quite a punch. I also have some relief supplies to help those who have been injured. Good luck, captain."`
+			`	You thank him for his assistance and the aid, and hurry to fire up your ship and return to human space. Hopefully Danforth's base has not already been overrun in your absence.`
```
Comment (jafdy) [inline data/wanderers middle.txt]: This addition reads a bit oddly to me.

File: data/wanderers middle.txt (hunk 1 above)
Comment (tehhowch) [reply data/wanderers middle.txt]: >Splits the log entry for the Quarg escorts (as some players choose instead to wander with them, and may forget exactly when they picked up these escorts).

The log bit about recruiting Quarg escorts is now journaled when you recruit them, as opposed to when you deliver them

File: data/wanderers middle.txt (hunk 2 above)
Comment (tehhowch) [reply data/wanderers middle.txt]: Suggestions welcomed.

`You thank him for his assistance, and for donating the aid. You hurry back to your ship to return to human space, hoping Danforth's base has not already been overrun in your absence.`?

File: data/wanderers middle.txt [hunk 3]
```diff
@@ -2027,14 +2027,18 @@ mission "Wanderers: Sestor: Quarg Help 2"
...
 	to offer
 		has "Wanderers: Sestor: Quarg Help 1: done"
 	on offer
 		conversation
 			`As you are coming in for a landing, you contact the Quarg and discover that Elias Hanover is indeed still living here. He meets you the moment you dock with the station, and you describe the situation. "This must be retaliation," he says. "They want to wipe Danforth and the Oathkeepers off the map to get revenge for him driving them out of their base on Avalon."`
-			`	He leaves to try to find help, and returns about fifteen minutes later. "I've got four Quarg ships who volunteered to help," he says. "That may not seem like a lot, but their ships pack quite a punch. Good luck, captain."`
-			`	You thank him for his assistance, and hurry to fire up your ship and return to human space. Hopefully Danforth's base has not already been overrun in your absence.`
+			`	He leaves to try to find help, and returns about fifteen minutes later. "I've got four Quarg ships who volunteered to help," he says. "That may not seem like a lot, but their ships pack quite a punch. I also have some relief supplies to help those who have been injured. Good luck, captain."`
```
Comment (Bladewood) [inline data/wanderers middle.txt]: `to help any injured`

Comment (endless-sky) [reply data/wanderers middle.txt]: Hanover doesn't necessarily have any personal wealth or supplies, so it might make more sense to just say, `And, they've offered some medical supplies as well, which they'll load onto your ship.`

File: data/wanderers middle.txt [hunk 4]
```diff
@@ -2105,6 +2108,10 @@ mission "Wanderers: Sestor: Bomb Zenith 1"
 			label end
 			`	Alarm sirens begin to sound. "Don't wait here to defend us," says Danforth. "Go straight to <planet> and deploy the device. It may be the only way you can save us."`
 				launch
+	on enter
+		dialog
+			`Danforth wasn't joking when he said that the device may be our only chance. There are almost a dozen Sestor capital ships in the system, and countless smaller craft tearing the Navy to shreds.`
```
Comment (endless-sky) [inline data/wanderers middle.txt]: It should be "*your* only chance." The narration is always second-person.

File: data/wanderers middle.txt [hunk 5]
```diff
@@ -2110,7 +2110,7 @@ mission "Wanderers: Sestor: Bomb Zenith 1"
 				launch
 	on enter
 		dialog
-			`Danforth wasn't joking when he said that the device may be our only chance. There are almost a dozen Sestor capital ships in the system, and countless smaller craft tearing the Navy to shreds.`
+			`Danforth wasn't joking when he said that the device may be your only chance. There are almost a dozen Sestor capital ships in the system, and countless smaller craft tearing the Navy to shreds.`
 			`Even the Quarg warships are having trouble with an opposing fleet this size. Perhaps a more powerful fleet could defeat the automata, but it would be a difficult battle.`
```
Comment (Bladewood) [inline data/wanderers middle.txt]: This paragraph should be indented.
